intel/init.py

```diff
@@ -176,7 +176,9 @@ def assign(cores, pool, count=None):
     free_cores = [c for c in cores if c.pool is None]
 
     # always prioritize free SST_BF cores, even if there may be none
-    free_cores.sort(key=lambda c: (c.is_sst_bf(), -c.core_id), reverse=True)
+    # moove up SST_BF cores AND keep order (spread or packet) among
```

Comment:
could you please fix the typo "moove" or unlock your branch so I can update it - thanks in advance

Reply:
Done